scripts/treedb_vectordbbench_artifact.py

```
@@ -728,6 +751,692 @@ def file_identity(path: Path) -> dict[str, Any]:
+        return None
+    if parsed.tzinfo is None:
+        errors.append(f"{label} must include a timezone")
+        return None
+    try:
+        return parsed.astimezone(_dt.timezone.utc)
+    except (ValueError, OverflowError):
+        errors.append(f"{label} must be an RFC3339 timestamp")
+        return None
+
+
+def _strict_json_loads(value: str) -> Any:
```

Ruff (0.16.2)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `_strict_json_loads`

(ANN401)